### ISSUE TITLE
Update data sources documentation

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#167]
## User story
1. As a {}
2. I want / need {}
3. So that {}

## Acceptance criteria
* Clarify / Simplify documentation of datasources
* Create table / spreadsheet with datasources and additonal information

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
